all_test.go

```diff
@@ -154,22 +155,23 @@ func isPrefix(prefix, str string) bool {
 }
 
 func TestIdentityCaseSensitivity(t *testing.T) {
+	t.Log("Testing case sensitivity")
 	c := setup1(t)
 
-	if e := c.CreateAuthenticatorIdentity("JOE", "123"); e == nil || !isPrefix(ErrIdentityExists.Error(), e.Error()) {
+	if e := c.CreateUserStoreIdentity("JOE", "123"); e == nil || !isPrefix(ErrIdentityExists.Error(), e.Error()) {
 		t.Errorf("CreateIdentity should fail because identities are case-insensitive. Instead, error is %v", e)
 	}
 	perm := Permit{}
 	roles := [2]byte{99}
 	perm.Roles = roles[:]
-	if e := c.SetPermit("JOE", &perm); e != nil {
+	if e := c.SetPermit(1, &perm); e != nil {
```

Comment:
These hardcoded user IDs should also be changed to symbols, the way it is done above, with variables such as joeUserID